scripts/node.sh

```diff
@@ -378,6 +378,15 @@ stn|stress|stressnet)
   dns_zone=stn.hmny.io
   syncdir=stn
   ;;
+p2p|p2pnet)
+  bootnodes=(
+    /ip4/52.40.84.2/tcp/9842/p2p/QmbPVwrqWsTYXq1RxGWcxx9SWaTUCfoo1wA6wmdbduWe29
```

Comment:
you can't use the same bootnode port 9842 with STN.  Please use a new one.

Reply:
Ah right, I just added it as a placeholder for now, Edgar's PR doesn't even use self-hosted bootnodes.

Now thinking of it, maybe this PR shouldn't even go in to master but only his PR.